core/src/main/java/io/aiven/klaw/controller/OperationalRequestsController.java

```diff
@@ -87,4 +119,32 @@ public ResponseEntity<List<OperationalRequestsResponseModel>> getConsumerOffsets
             isMyRequest),
         HttpStatus.OK);
   }
+
+  @RequestMapping(
+      value = "/operationalRequests/approver",
```

Comment:
I think that 
"/operationalRequests" with a path param isApproval

then if(isApproval) {
return getApprovals;
} else {
return myTeamRequests
}


Then you can remove the myTeamRequests endpoint as well. I think semantically it reads well for the api.

Reply:
I tried this initially, but not all the req parameters are same for both. So had to create a different one

Reply:
"/operationalRequests" for myTeamRequests then?
"/operationalRequests/approval" for approvers?

Reply:
Updated, using your earlier suggestion. Made it one endpoint. 
/operationalRequests/requestsFor/{requestsFor}